pylintrc

```diff
@@ -188,7 +194,7 @@ notes=TMP,XXX
 # List of decorators that produce context managers, such as
 # contextlib.contextmanager. Add to this list to register other decorators that
 # produce valid context managers.
-contextmanager-decorators=contextlib.contextmanager
+contextmanager-decorators=contextlib.contextmanager,closing
```

Comment:
Please put these on separate lines, like above because this list is going to grow fast.

Reply:
:+1: done